python-api-examples/offline-decode-files.py

```
@@ -527,18 +545,23 @@ def main():
         streams.append(s)
 
     recognizer.decode_streams(streams)
-    results = [s.result.text for s in streams]
     end_time = time.time()
     print("Done!")
 
-    for wave_filename, result in zip(args.sound_files, results):
-        print(f"{wave_filename}\n{result}")
+    for wave_filename, stream in zip(args.sound_files, streams):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)